adobe/react-spectrum
# fix(#2599): TableView: space key on link within cell scrolls the TableView down a page

Closes #2599

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 2559](https://github.com/adobe/react-spectrum/issues/2599).

## 📝 Test Instructions:
1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/903179155a85c47707c9bfb6902e0deef1b7d892/storybook/index.html?path=/story/tableview--async-loading
2. Tab to move focus to the first article link in the first row within the table body.
3. Press Space key.
4. Since this is a link, Space shouldn't activate the link, but the event should bubble to be handled by the table row, toggling selection. `preventDefault` should be called on the event to prevent the TableView from scrolling.

## 🧢 Your Project:

Adobe/Accessibility
